queries.yaml

```diff
@@ -650,19 +650,42 @@ post_process:
       start_zoom: 8
       end_zoom: 15
 
+  # no lake labels at zoom 0-3:
+  # https://github.com/tilezen/vector-datasource/issues/1730
+  - fn: vectordatasource.transform.drop_properties
+    params:
+      source_layer: water
+      start_zoom: 0
+      end_zoom: 4
+      properties: [name]
+      geom_types: [Polygon, MultiPolygon]
+      where: >-
+        kind == 'lake'
+
   # before extracting the label placement points, remove small water areas which
```

Comment:
I find this comment confusing, perhaps reword to:

>   # before extracting the label placement points, remove names 
>   # from small water areas which are probably too small to label.

Reply:
Yup, sorry. I've expanded the comment and added context in 349cffa. Please take a look and let me know if that makes it clearer.